examples/go-samples/containerized/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine
+
+# Add JSON support to Helm
+# helm plugin install https://github.com/Microsoft/helm-json-output --version master
+#
+# Now install your app ...
+# helm json install stable/rabbitmq
+#
+# Here's example output hardcoded for illustration ...
```

Comment:
Add: the helm.sh will return a fake RabbitMQ helm release if run with `docker run install rabbitmq` (coiled-gecko) and a fake MySQL (happy-squirrel) otherwise

?